react/components/ProductImages/components/Carousel/index.js

```diff
@@ -259,7 +259,7 @@ class Carousel extends Component {
             bulletSelector: `.swiper-pagination-bullet`,
             bulletClass: `swiper-pagination-bullet ${cssHandles.swiperBullet}`,
             bulletActiveClass:
-              'c-action-primary swiper-pagination-bullet-active',
+              `c-action-primary swiper-pagination-bullet-active ${applyModifiers(cssHandles.swiperBullet, 'active')}`,
```

Comment:
Now there is an eslint problem 😄

Reply:
You can guide me to fix the lint problem?  I run the yarn command but nothing changes. When I installed the eslint plugin the entire file produces error...

Reply:
After you run `yarn` in the root folder it will install all eslint dependencies, so if you open in vscode and have eslint and prettier installed in vscode it will show the error to you, so you just have to press `Ctrl + .` or `cmd + .` and select the first option to prettier fix it

Reply:
Finally, I get it hahaha, my vscode happened to be with stranges configurations. Thanks!